recipes/pygaze/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pygaze" %}
+{% set pypiname = "python-" + name %}
+{% set version = "0.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz"
+  sha256: "e277d55cbe12dfc4d48333696e3e8b9fff0e549ec273fda3add69e8999d10295"
+
+build:
+  skip: true  # [py2k]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 0
```

Comment:
Could this one be noarch python too?

Reply:
Noarch packages apparently don't allow selectors. Since this package doesn't build on Python 2.7 (for not very obvious reasons) I had to skip the build for this platform with the py2k selector.
Is there another way to make a package noarch, but not use selectors?

Reply:
You can skip python versions by using `python >=3` in noarch packages

Reply:
Never knew, learnt something again. I made these changes and just pushed them.